### DISCUSSION TITLE
问下各位，tx现在对QQ个人账号机器人封禁力度大吗

### DISCUSSION THREAD

**Discussion by Alexshaneeee** [nonebot/discussions#475] [通用]
去年用nonebot1写了一个简单的自用机器人，登录自己QQ小号部署在华为云买的一台ECS上，后来有段时间老被封禁，加上服务器快到期了就不搞了。现在想重新回来弄一个，但是tx已经收编了机器人功能推出了自己的官方机器人，还没法个人资质申请，那现在还可以用自己小号登录的办法吗，封禁力度如何

**Comment by yanyongyu** [comment]
目前你可以使用 Lagrange、LLOneBot、NapCat 等方法 在不违规的情况下稳定登录 QQ

**Reply by Alexshaneeee** [reply]
thanks，我试试
